modules/services/improveOSM.js

```diff
@@ -210,12 +210,13 @@ export default {
                                 // One-ways can land on same segment in opposite direction
                                 loc = preventCoincident(loc, false);
 
-                                var d = new iOsmError({
+                                var d = new qaError({
+                                    // Info required for every error
                                     loc: loc,
-                                    comments: null,
-                                    error_subtype: '',
+                                    service: 'improveOSM',
                                     error_type: k,
-                                    icon: 'fas-long-arrow-alt-right',
```

Comment:
Can you add this icon back?  It got missed because I only added it last night  😆

Reply:
or are this in data/qa_errors.json now?  thats ok too..

Reply:
Yeah this has been moved to the .json

I figure values we're setting in iD and not those being read from the service API can go there.